src/main/java/ch/njol/skript/expressions/ExprCoordinate.java

```diff
@@ -32,19 +28,25 @@
 import ch.njol.skript.lang.Expression;
 import ch.njol.skript.lang.SkriptParser.ParseResult;
 import ch.njol.util.Kleenean;
+import org.bukkit.Chunk;
+import org.bukkit.Location;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
 
 /**
  * @author Peter Güttinger
  */
 @Name("Coordinate")
-@Description("Represents a given coordinate of a location. ")
+@Description("Represents a given coordinate of a location/chunk. ")
 @Examples({"player's y-coordinate is smaller than 40:",
-		"	message \"Watch out for lava!\""})
-@Since("1.4.3")
```

Comment:
This shouldn't be removed
Also, it should be updated
`@Since("1.4.3, INSERT VERSION (chunks)")`

Reply:
Okay, so wherever you can cram Final? Otherwise, thank you

Reply:
Yes, we no longer use final modifier on method params or variables, unless it's really needed